src/sass/components/_nested-navigation.scss

```diff
@@ -0,0 +1,127 @@
+.#{$prefix}-nested-nav {
+  &__list {
+    flex-basis: 100%;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  &__list[hidden],
+  &__list[aria-hidden='true'] {
+    display: none;
+  }
+
+  &__item {
+    align-items: center;
+    border-top: 1px solid $color-black--100;
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 0; // resets Rivet default margin-top
+  }
+
+  &__link {
+    color: $color-black;
+    flex-grow: 1;
+    padding: $sm;
```

Comment:
The overall height of each nav item is looking a bit tall to me once you look at it in the browser. I bumped the `$sm` value down to a slightly small magic number and it felt much better:

```scss
padding: $sm*.875;
```